cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```
On POSIX-compliant systems, `/bin/sh` links to whatever the default shell is on that system, so I think making it `sh` instead of `bash` will keep it working in case someone is working on a non-BASH shell (likely someone on a Mac, whose default shell is ZSH)